apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-description/style.scss

```diff
@@ -11,6 +12,29 @@
 			background-color: transparent;
 		}
 	}
+
+	.wp-block {
+		&.is-selected {
+			.wp-block-a8c-site-description {
+				&::-webkit-input-placeholder {
```

Comment:
Could potentially add a mixin for this rather than duplicating between blocks

Reply:
I would say probably worth it if we have time but not that important at the moment?

Reply:
... yeh, if we don't do it now it probably won't happen - are you happy with folder/file layout added now to handle this

Reply:
> if we don't do it now it probably won't happen
so true. Looks good to me!